packages/docusaurus-search-local/src/server/index.ts

```diff
@@ -456,6 +469,15 @@ export const tokenize = (input) => lunr.tokenizer(input)
           };
         });
 
+      logger.info(`Ignored pages: ${[...actuallyIgnored].join(", ")}`);
+
+      if (initialIgnore.size)
```

Comment:
nitpick: Could you add `{` and `}` to the if statement and remove the blank line between it and the logger call?